.docksal/etc/conf/settings.kanopi.php

```diff
@@ -1,15 +1,26 @@
 <?php
 
+// phpcs:ignoreFile
+
 /**
  * @file
  * Local settings that are copied via fin init.
  */
 
-// Docksal DB connection settings.
+// Docksal Default DB connection settings.
 $databases['default']['default'] = [
```

Comment:
I think we can remove this also as the `.docksal/etc/php/prepend.php` contains the default configuration details as environment variables.

Reply:
This is in our current configuration which is why it remains in this update.
If we want to remove it from the `drupal-stater` then maybe we should log a ticket to do so outside of what this pull request is trying to accomplish.

That said, most developers would expect this to exist and it might cause confusion so when we do remove it we need to make sure to add a comment referencing the change we made to the "Drupal Way" of handling the database variable.